CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,21 @@
+# Community Guidelines
+
+A safe, respectful, productive and collaborative environment is important for the boardgame.io community. These guidelines apply to all community communications channels (such as the official Gitter group, GitHub issues and pull requests etc.).
+
+1. Be welcoming to all. Help newcomers with questions about the project (even if these are answered elsewhere).
+ 
+1. Do not make personal attacks or disparaging personal remarks.
+ 
+1. When interpreting the words and actions of others, assume good faith and intentions.
+ 
+1. Do not engage in harassing or bullying behavior of any kind.
+
+## What to do if you see a violation of these guidelines
+
+You may do either or both of the following:
+
+1. Politely message the perpetrator to steer them in the right direction.
+
+1. Contact a moderator privately (or email moderators@boardgame.io) to report bad behavior and request intervention.
+
+Moderators will encourage better behavior or issue a warning as appropriate. Further action (like banning a member) may be necessary as a last resort when other measures fail.
```

Comment:
Thanks for putting this together!

Should we specify who the current moderators are here? I'm assuming it would be me and you for now, but it would be nice to clarify who receives emails sent to the moderators address.

Reply:
Sure, that makes sense.